libminifi/include/core/state/FlowIdentifier.h

```diff
@@ -68,12 +68,7 @@ class FlowIdentifier {
     bucket_id_ = other.bucket_id_;
     flow_id_ = other.flow_id_;
   }
-  FlowIdentifier &operator=(const FlowIdentifier &other) {
-    registry_url_ = other.registry_url_;
-    bucket_id_ = other.bucket_id_;
-    flow_id_ = other.flow_id_;
-    return *this;
-  }
+  FlowIdentifier &operator=(const FlowIdentifier &other) = default;
```

Comment:
Please `=default` the "copy ctor" above as well and remove its `explicit` specifier.

Reply:
As requested.